README.md

```diff
@@ -6,132 +6,57 @@ Members of the community are rebooting this library under the guidance of the Xa
 
 ## Build Status
 
-If you like to live dangerously (and while we are preparing our v1), you can use our [nightly](https://pkgs.dev.azure.com/xamarin/public/_packaging/XamarinCommunityToolkitNightly/nuget/v3/index.json) feed to try out packages right now.
+If you like to live dangerously, you can use our [nightly](https://pkgs.dev.azure.com/xamarin/public/_packaging/XamarinCommunityToolkitNightly/nuget/v3/index.json) feed to try out the latest and greatest.
 
 | Build Server | Type         | Platform | Status |
 |--------------|--------------|----------|--------|
-| Azure DevOps         | Build        | Windows & Mac  | [![Build Status](https://dev.azure.com/xamarin/public/_apis/build/status/xamarin/CommunityToolkit/xamarin.XamarinCommunityToolkit%20(Public)?branchName=main)](https://dev.azure.com/xamarin/public/_build?definitionId=55&_a=summary)                                                  |
+| Azure DevOps         | Build        | Windows & Mac  | [![Build Status](https://dev.azure.com/xamarin/public/_apis/build/status/xamarin/CommunityToolkit/xamarin.XamarinCommunityToolkit%20(Public)?branchName=main)](https://dev.azure.com/xamarin/public/_build?definitionId=55&_a=summary) |
 
 ## Sample App
 
-Browsing the [sample app](./samples) is the best place to start exploring what's available today. 
+Browsing the [sample app](./samples) is the best place to start exploring what's available today.
 
 ## Installation
 
-The toolkit is available via NuGet, and should be installed into all your projects:
+The Toolkit is available via NuGet, and should be installed into all (shared and each platform) your projects:
```

Comment:
> and should be installed into all (shared and each platform) your projects

"and should be installed into all of your projects (shared and individual platforms)"